management_api_app/services/health_checker.py

```diff
@@ -10,11 +12,24 @@ def create_state_store_status() -> (StatusEnum, str):
     status = StatusEnum.ok
     message = ""
     try:
-        client = CosmosClient(STATE_STORE_ENDPOINT, STATE_STORE_KEY)    # noqa: F841 - flake 8 client is not used
+        primary_master_key = get_store_key()
+        client = CosmosClient(config.STATE_STORE_ENDPOINT, primary_master_key)    # noqa: F841 - flake 8 client is not used
     except exceptions.ServiceRequestError:
         status = StatusEnum.not_ok
         message = strings.STATE_STORE_ENDPOINT_NOT_RESPONDING
     except:     # noqa: E722 flake8 - no bare excepts
         status = StatusEnum.not_ok
         message = strings.UNSPECIFIED_ERROR
     return status, message
+
+
+def get_store_key() -> str:
```

Comment:
This looks like a dupe of the same function in management_api_app/api/dependencies/database.py - can we remove this and import that one instead? or did I miss some difference?

Reply:
true, fixed